core/services/eth/log_broadcaster.go

```diff
@@ -48,8 +49,8 @@ type logBroadcaster struct {
 	ethClient     Client
 	orm           orm
 	backfillDepth uint64
-	connected     bool
-	started       bool
+	connected     *abool.AtomicBool
+	started       *abool.AtomicBool
```

Comment:
Not sure I recall. I think it prevented blocking if you accidentally call Stop twice or something?

Reply:
Ah, in the past we've used `sync.Once` for that.  Probably cleaner, although I'd argue that calling Start/Stop more than once should actually cause a panic, as it's a programming error.

Reply:
Ah, it was in case you called Stop and it wasn't started already.

There was definitely some sort of test that needed this. I'll leave it here for now because I suspect I may introduce flakiness by removing it.